### PR TITLE
Add kure-potlin to marketplace

### PR DESCRIPTION
**Pull request by schalkms**

### CODE REVIEW

File: website/src/data/marketplace.js [hunk 1]
```diff
@@ -110,6 +110,27 @@ export const extensions = [
...
+      "LoopDefinition",
+      "ReturnStatement",
+      "VariableDefinition",
+      "ReturnUnit",
+      "ClassDefinition",
+      "AbstractClassDefinition",
+      "ThrowExpression",
+      "MutableCollections",
+      "BranchStatement",
+      "MissingElse"
```
Comment (cortinico) [inline website/src/data/marketplace.js:130]: Can we sort them?

File: website/src/data/marketplace.js [hunk 2]
```diff
@@ -110,6 +110,27 @@ export const extensions = [
     usesTypeResolution: false,
     tags: ["ruleset"],
   },
+  {
+    title: "Kure",
+    description:
+      "The purpose of this ruleset for functional programming is to report the potential use of impure language elements in kotlin code.",
+    repo: "https://github.com/neeffect/kure-potlin",
+    tags: ["plugin", "ruleset"],
```
Comment (cortinico) [inline website/src/data/marketplace.js:118]:
```suggestion
    tags: ["ruleset"],
```

**Review by cortinico — COMMENTED** [review]
Thanks for adding this ruleset 👍

File: website/src/data/marketplace.js (hunk 1 above)
Comment (schalkms) [reply website/src/data/marketplace.js:130]: Hmm, sure, but all the other rulesets don't apply any sorting order. I copied it them from the readme file in the original repo.

Comment (schalkms) [reply website/src/data/marketplace.js:130]: Hmm, sure, but all the other rulesets don't apply any sorting order. I copied them from the readme file in the original repo.

Comment (schalkms) [reply website/src/data/marketplace.js:130]: I could work over the whole file and submit a separate PR. I think that would be easier.

Comment (cortinico) [reply website/src/data/marketplace.js:130]: Yup that's not a blocker 👍

**Review by cortinico — APPROVED** [review]

**Pull request merged**